mempool/clist_mempool.go

```diff
@@ -291,7 +291,7 @@ func (mem *CListMempool) LaneSizes(lane LaneID) (numTxs int, bytes int64) {
 
 	txs, ok := mem.lanes[lane]
 	if !ok {
-		return 0, bytes
+		panic(fmt.Errorf("lane %s not found while fetching lane size", lane))
```

Comment:
Can it happen that we give it the wrong lane?

Reply:
That would be a bug. And if we pass a wrong but existing lane, it won't panic.